docs/users/installation.md

```diff
@@ -87,17 +113,71 @@ git diff // should produce a diff
 
 ### Settings and tasks
 
-| Name               | Type           | Description                                                                                                                      |
-| ------------------ | -------------- | -------------------------------------------------------------------------------------------------------------------------------- |
-| `scalafix <args>`  | `Unit`         | Invoke scalafix command line interface directly. See [--help](#help) use tab completion to explore supported arguments           |
-| `scalafixConfig`   | `Option[File]` | .scalafix.conf file to specify which scalafix rules should run. Defaults to `.scalafix.conf` in the root directory if it exists. |
+| Name              | Type                       | Description                                                                                                                       |
+| ----------------- | -------------------------- | --------------------------------------------------------------------------------------------------------------------------------- |
+| `scalafix <args>` | `TaskKey[Unit]`            | Invoke scalafix command line interface directly. Use tab completion to explore supported arguments or consult [--help](#help)     |
+| `scalafixConfig`  | `SettingKey[Option[File]]` | .scalafix.conf file to specify which scalafix rules should run. Defaults to `.scalafix.conf` in the root directory, if it exists. |
+
+### Main and test sources
+
+The task `myproject/scalafix` runs only in the project `myproject` for main
+sources, excluding test sources. To run Scalafix on test source, execute
+`myproject/test:scalafix` instead. To run on both main and test sources, execute
+`all myproject/scalafix myproject/test:scalafix`
```

Comment:
I never seen `all`. What does it do? Run two task in parallel?

Reply:
Yes, it's like executing two tasks at once.